### PR TITLE
Update RSVP button to indicate that registration is closed

### PR DESCRIPTION
**Pull request by dsdude123**

### CODE REVIEW

**Comment by SaamIAm** [comment]
LGTM

File: index.html [hunk 1]
```diff
@@ -51,7 +51,7 @@ <h3>April 19, 2019</h3>
         </div>
         <div class="register">
             <a href="https://goo.gl/forms/DWhnBaRRVMHm0wTL2" class="nav-register">
```
Comment (Chris-Johnston) [inline index.html]:
```suggestion
            <a href="#" class="nav-register">
```
Let's just remove the link.

File: index.html [hunk 2]
```diff
@@ -65,7 +65,7 @@ <h3>Friday <b>April 19th</b>, 2019 from <b>9 AM to 10PM</b> in the North Creek E
                     Center & 2nd floor of the ARC
                 </h3>
                 <a href="https://goo.gl/forms/DWhnBaRRVMHm0wTL2" class="top-fold__register">
```
Comment (Chris-Johnston) [inline index.html]:
```suggestion
                <a href="#" class="top-fold__register">
```

File: index.html [hunk 3]
```diff
@@ -66,7 +66,7 @@ <h3>Friday <b>April 19th</b>, 2019 from <b>9 AM to 10PM</b> in the North Creek E
                 </h3>
                 <div class="register-buttons">
                     <a href="https://goo.gl/forms/DWhnBaRRVMHm0wTL2" class="top-fold__register">
```
Comment (Chris-Johnston) [inline index.html]:
```suggestion
                    <a href="#" class="top-fold__register">
```
merge broke prev review

**Comment by dsdude123** [comment]
Removed links and also fixed a button that said "Register Closed!" rather than "Registration Closed!"

**Pull request merged**